src/Sylius/Bundle/ApiBundle/Exception/PaymentMethodCannotBeChangedException.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ApiBundle\Exception;
+
+use ApiPlatform\Core\Exception\InvalidArgumentException;
+
+final class PaymentMethodCannotBeChangedException extends InvalidArgumentException
```

Comment:
I believe it should extends `\RuntimeException`, @NoResponseMate @jakubtobiasz, wdyt?
```suggestion
final class PaymentMethodCannotBeChangedException extends \RuntimeException
```

Reply:
I always extend `\Exception` 🤷🏼‍♂️. In this case, I'd go with `RuntimeException` or just `Exception`.